pkg/agent/plugin/workloadattestor/k8s/k8s.go

```diff
@@ -757,3 +787,45 @@ func newCertPool(certs []*x509.Certificate) *x509.CertPool {
 	}
 	return certPool
 }
+
+func newConfigFromHCL(hclConfig *SigstoreHCLConfig, log hclog.Logger) *sigstore.Config {
```

Comment:
looks like the config is the same for k8s and docker, may we move this to a common place? and put all validations together?